packages/core/lib/version.js

```diff
@@ -60,9 +60,15 @@ const logWeb3 = (logger = console) => {
   logger.log(`Web3.js v${web3Version}`);
 };
 
+const logGanache = (logger = console) => {
+  const ganacheVersion = pkg.dependencies.ganache;
+  logger.log(`Ganache V${ganacheVersion}`);
```

Comment:
one more nit, can you make the `V` lowercase so it matches the format that we print web3's version in?

Reply:
okie I will make the change.